depscan/lib/csaf.py

```diff
@@ -1083,14 +1082,14 @@
 }
 
 TOML_TEMPLATE = (
-    "https://raw.githubusercontent.com/owasp-dep-scan/dep-scan/master/contrib"
-    "/csaf.toml"
+    "https://raw.githubusercontent.com/owasp-dep-scan/dep-scan/master"
```

Comment:
Can we bundle this in the project and use file write to create it rather than download it? That way, this can work offline in air-gapped environments, too.

Reply:
@prabhu Yes, I had originally done this because I couldn't get tomli-w to output with user-friendly formatting. However, I then ran into some related issues and have reverted to using toml instead.

The only thing about not downloading the file is that no comments will be included on the toml. I guess that's ok, but in that case, they will certainly need to download the csaf readme to understand what's going on.

Reply:
@prabhu If you mean to include the actual toml file in the package, I can't seem to get that to happen. Not sure what I need to put in the pyproject.toml. If you know how, I'm all ears!

Reply:
Does having an entry in MANIFEST.in work? Wondering if the license data are still getting bundled into the package since our switch to pyproject.

https://github.com/owasp-dep-scan/dep-scan/blob/master/MANIFEST.in

Reply:
@prabhu The license data is included because the vendor directory is included as a package in the pyproject.toml, whereas contrib is not.

To include non-python files, appears the non-python files have to be in a module AND specified in the manifest. 

So we can either add contrib as a package to the pyproject.toml, or perhaps create a module named depscan_data to house the vendor data as well as the csaf.toml.

Reply:
module for csaf data sounds good to me. or can the entire toml be placed in a template string?

Reply:
@prabhu Either works. I'll put it in a template string for now and we can change later if it becomes too long or complex to manage. The comments won't be there, but they would be gone anyway after the first update. One thing is that a number of fields are enums per the csaf spec, so people will certainly need to reference our documentation to know what the valid values are, or to even know they can't just put whatever. I want to add validation around this and other fields, but this is a reasonable stopping point for the first iteration.